spec/models/king_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  describe '#create' do
+    it 'creates a king in the database' do
+      king = FactoryGirl.create(:king)
+      expect(king.type == 'King').to eq(true)
+    end
+  end
+
+  describe '#valid_move?' do
+    it 'returns true if the move is diagonal up-right one square' do
+      king = FactoryGirl.create(:king, x_position: 4, y_position: 4)

```

Comment:
OK, good idea. So in `king_spec.rb`, in each test, the line would be shortened to: `king = FactoryGirl.create(:king)` . . . correct?